doc/getting-started/up2.rst

```diff
@@ -75,21 +75,22 @@ You will need to keep these in mind in a few places:
   .. code-block:: none
 
      # efibootmgr -c -l "\EFI\acrn\acrn.efi" -d /dev/mmcblk0 -p 1 -L "ACRN Hypervisor" \
-         -u "bootloader=\EFI\org.clearlinux\bootloaderx64.efi uart=mmio@0x9141e000"
+         -u "bootloader=\EFI\org.clearlinux\bootloaderx64.efi uart=bdf@0:18.1"
 
 UP2 serial port setting
 =======================
 
-The serial port in the 40-pin HAT connector is located at ``MMIO 0x0x9141e000``.
-You can check this from the ``dmesg`` output from the initial Clearlinux installation.
+The serial port in the 40-pin HAT connector is located at ``serial PCI BDF 0:18.1``.
+You can check this from the ``lspci`` output from the initial Clearlinux installation.
 
 .. code-block:: none
 
-   # dmesg | grep dw-apb-uart
-   [2.150689] dw-apb-uart.8: ttyS1 at MMIO 0x91420000 (irq = 4, base_baud = 115200) is a 16550A
-   [2.152072] dw-apb-uart.9: ttyS2 at MMIO 0x9141e000 (irq = 5, base_baud = 115200) is a 16550A
+   #lspci | grep UART
+00:18.0 . Series HSUART Controller #1 (rev 0b)
+00:18.1 . Series HSUART Controller #2 (rev 0b)
```

Comment:
These two lines need to start at the same indentation as the `#lspci...`

Reply:
Got. thanks!